backend/routes/web.php

```diff
@@ -14,7 +18,33 @@
 */
 
 Route::get('/', function () {
-    return ['Laravel' => app()->version()];
+    $following_user = User::find(6)->following;
+
+    $ids = [];
+    foreach($following_user as $user)
+    {
+        $ids[] = $user->id;
+    }
+    array_push($ids, 6);
+
+    $user_activities = ActivityLog::whereIn('user_id', $ids)->orderBy('created_at', 'desc')->get();
+
+    foreach($user_activities as $activity)
+    {
+        $name = User::find($activity->user_id)->name;
+
+        if ($activity['activity_type'] === 'Follow')
+        {
+            $name .= ' followed ' . User::find($activity->activity_id)->name;
+        }
+        if ($activity['activity_type'] === 'Lesson')
+        {
+            $name .= ' learned ' . Category::find($activity->activity_id)->title;
+        }
+        dump($name);
+    }
+
+    // return 
```

Comment:
Can this be transferred into a controller? Also, you're not returning any value here

Reply:
I forgot to remove the testing part in the web.php in laravel